config/initializers/event_store.rb

```diff
@@ -1,10 +1,13 @@
 # 0.9.0 version of rails_event_store gem causes active_record to load pre-maturely
-ActiveSupport.on_load(:active_record) do
-  require 'rails_event_store'
-  require 'three_scale/sidekiq_retry_support'
 
-  System::Application.configure do
-    config = self.config
-    config.event_store = EventStore::Repository.new
+Rails.application.config.to_prepare do
```

Comment:
Same as in comment above. I'm not sure combining `Rails.application.config.to_prepare` and `ActiveSupport.on_load(:active_record)` is a good idea.

Reply:
I am not sure either :)

Reply:
This is for fixing `db:create` task:

```
$ RAILS_ENV=test bundle exec rake ci:db:ready db:create
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
Connected to mysql2://root@127.0.0.1/systemdb0
rake aborted!
NameError: uninitialized constant EventStore

    config.event_store = EventStore::Repository.new
                         ^^^^^^^^^^
Did you mean?  RubyEventStore
/home/dmayorov/Projects/3scale/porta/config/initializers/event_store.rb:9:in `block (2 levels) in <main>'
/home/dmayorov/Projects/3scale/porta/config/initializers/event_store.rb:7:in `block in <main>'
/home/dmayorov/Projects/3scale/porta/config/initializers/event_store.rb:3:in `<main>'
/home/dmayorov/Projects/3scale/porta/config/environment.rb:8:in `<main>'
/home/dmayorov/.asdf/installs/ruby/3.1.5/bin/bundle:25:in `load'
/home/dmayorov/.asdf/installs/ruby/3.1.5/bin/bundle:25:in `<main>'
Tasks: TOP => db:create => db:load_config => environment
(See full trace by running task with --trace)
```